### PR TITLE
feat(plugin): support private repository

### PR DESCRIPTION
**Pull request by yaeda**
* Add 'private' property (default: false)
* When 'private' is true, source contents are copied/published to the plugin directory (/gitbook/gitbook-plugin-github-issue-feedback/contents/).
  Plugin load contents from there instead of the Github API.

How do you think?

### CODE REVIEW

File: src/plugin.js [hunk 1]
```diff
@@ -29,11 +35,19 @@ function getResourceURL(config, filePath, branch) {
...
     return urlJoin(`https://github.com/`, config.repo, `edit`, branch, filePath);
 }
-function getAPIURL(config, filePath) {
-    if (config["githubAPIBaseURL"]) {
-        return urlJoin(config["githubAPIBaseURL"], filePath);
+function getContentURL(config, filePath) {
+    if (config.private) {
+        return urlJoin(
+            location.origin,
+            "gitbook/gitbook-plugin-github-issue-feedback/contents",
```
Comment (azu) [inline src/plugin.js:42]: https://github.com/azu/gitbook-plugin-github-issue-feedback/pull/5/files#diff-168726dbe96b3ce427e7fedce31bb0bcR27 use `pkg.name`.
But this line use hardcode value.

We should unify usage.

1. `pkg.name`
2. hardcode
3. create shared `constants.json`

**Review by azu — CHANGES_REQUESTED** [review]
Thanks for PR.
I can not test it. Because I have not private repository :)
But, Looks good to me.

We need to update README too.
https://github.com/azu/gitbook-plugin-github-issue-feedback#usage

File: src/plugin.js (hunk 1 above)
Comment (yaeda) [reply src/plugin.js:42]: Thank you for your review.

I'd like to choose option 2 (hard-code) in this PR. There are some other hard-coded strings. I think it is better to create another PR/Issue for refactoring them.

**Review by azu — APPROVED** [review]
LGTM

**Comment by azu** [comment]
Thanks.

I've released [1.3.0](https://github.com/azu/gitbook-plugin-github-issue-feedback/releases/tag/1.3.0 "1.3.0")

**Comment by yaeda** [comment]
Sorry for my late action and Thank you very much! 😄

**Pull request merged**